panel/panel.js

```diff
@@ -104,9 +105,13 @@ export default class ComponentsPanel extends StacheElement {
 
 			selectedNode: type.maybeConvert(ObservableObject),
 			componentTreeError: String,
+			tagName: {
```

Comment:
`get default () {}` is only necessary when returning objects. Also if you just do `tagName: ""` it will automatically infer the type.